crates/core/c8y_api/src/json_c8y.rs

```diff
@@ -153,9 +153,14 @@ impl TryFrom<ThinEdgeEvent> for C8yCreateEvent {
                 time = OffsetDateTime::now_utc();
                 extras = HashMap::new();
             }
-            Some(event_data) => {
+            Some(mut event_data) => {
                 text = event_data.text.unwrap_or_else(|| event_type.clone());
                 time = event_data.time.unwrap_or_else(OffsetDateTime::now_utc);
+
+                // TODO: Not yet clear how to address "type"
+                if let Some(new_type) = event_data.extras.remove(".type") {
```

Comment:
We should only used `type` not `.type` (sorry for the confusion from the other ticket).

```suggestion
                if let Some(new_type) = event_data.extras.remove("type") {
```

Reply:
fixed :D